roles/docker/meta/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 galaxy_info:
   author: your name
```

Comment:
git blame tells me the following

    ^5994dee (Kenny Woodson   2014-09-16 13:15:48 -0400 1) ---